docs/templates/getting-started/search_spaces.md

```diff
@@ -73,6 +73,10 @@ The stochastic expressions currently recognized by hyperopt's optimization algor
 * `hp.uniform(label, low, high)`
   * Returns a value uniformly between `low` and `high`.
   * When optimizing, this variable is constrained to a two-sided interval.
+  
+* `hp.uniformint(label, low, high)`
+  * Returns an integer uniformly between `low` and `high`.
```

Comment:
are low and high included?

Reply:
I think hp.uniformint(label, high) is also valid, and then would between 0 and high. Can you please check and rewrite this?